.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Gem release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.2' # Not needed with a .ruby-version file
+
+      - name: Install dependencies
+        run: npm install && bundle install
+
+      - name: Build and validate gem
+        run: gem build
```

Comment:
should it be `gem build` or `gem build phraseapp-in-context-editor-ruby.gemspec` (from the deleted RELEASING.md)? we don't need to do `gem push` anymore? 

https://github.com/Gusto/semantic-release-rubygem offers steps for `prepare` and `publish` 🤔

Reply:
`gem build` does the same as I can see.

I think the steps are doing right things by default, no config needed, but tbh I'm not sure. I'm not sure if we need to change them manually 🤔 The readme in the repo was a bit confusing to me

Reply:
Same - the README in the repo was confusing and didn't provide much documentation on what to do imo.

I saw this setup: https://tianhaozhou.medium.com/ruby-semantic-release-github-actions-efa5db43332a but I'm confused where they build the gem 🤔  It seems to be automatically taken care of by the `semantic-release-rubygem` plugin

I think we'll just have to try it out and see 🤞